include/helib/query.h

```diff
@@ -515,6 +603,126 @@ class QueryBuilder
 
     return std::move(convertStack.top());
   }
+
+  /**
+   * @brief Given a vecvec where outer clauses correspond to ANDs and inner
+   * clauses correspond to ORs, deletes duplicated columns and columns for which
+   * both positive and negation occur from inner clauses, and deletes empty
```

Comment:
"columns for which positive occur from inner clauses"?
Not sure what that means.

Reply:
These descriptions were hard to word, basically where you have (column i or not column i) in an inner clause it will remove both

Reply:
Ah I see, then maybe add that to the description to have something like 
"and columns ... from inner clauses i.e. (column i OR NOT column i)"

Reply:
I've updated this here https://github.com/helibproject/HElib/blob/67fc4872c6d1b56394693a5f73a64e46a2606bdb/include/helib/query.h#L609